java11_debian/Dockerfile

```diff
@@ -1,4 +1,9 @@
-FROM openjdk:11.0.5-jdk-slim-buster
+FROM openjdk:11.0.11-jdk-slim-buster
+
+RUN apt-get update
```

Comment:
This is unneeded. It's done in the first part of the `RUN` instruction on line 9. We do it all in one instruction to avoid creating an extra layer for the docker image.